packages/phase2cli/types/index.ts

```diff
@@ -0,0 +1,219 @@
+import { FirebaseApp } from "firebase/app"
+import { DocumentData, DocumentReference, Firestore } from "firebase/firestore"
+import { Functions } from "firebase/functions"
+import { User as FirebaseAuthUser } from "firebase/auth"
+
```

Comment:
Comments on types (just open an issue don't fix)

Reply:
Tracked in #213